packages/cloud_firestore/cloud_firestore/example/pubspec.yaml

```diff
@@ -12,6 +12,8 @@ dependencies:
 dependency_overrides:
   firebase_core:
     path: ../../../firebase_core/firebase_core
+  cloud_firestore_platform_interface:
```

Comment:
@Salakar this was necessary to make the functionality available.

Reply:
This is fine, thanks for adding. 

I am working on some additional CI checks/tooling to check things like this in future.